tinygrad/engine/realize.py

```diff
@@ -117,6 +117,9 @@ def __init__(self, total_sz, dest_device, src_device):
     else: name = f"{type(self).__name__[6:].lower()} {total_sz:8d}, {dest_device[:7]:>7s} <- {src_device[:7]:7s}"
     super().__init__(colored(name, "yellow"), dest_device, 0, total_sz)
   def copy(self, dest, src):
+    if src.device.startswith("DISK") and dest.device == "IOS": #TODO, hack for iOS
+      dest.allocator.copy_from_disk(dest,src)
+      return
```

Comment:
Hack?

Reply:
this is needed to be able to copy weights from iOS's bundle to metalbuffers, it made testing this faster, but won't be needed for exo. Can just send weights over the network (which this does if the file isn't found) and then cache them

Reply:
I've removed this, it uses the same path as metal now